scripts/prepdocs.ps1

```diff
@@ -1,35 +1,69 @@
-Write-Host ""
```

Comment:
Do we need to update the corresponding .sh script as well?

Reply:
No, this was just a pwsh issue.